testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -485,4 +480,245 @@ private void testSpecialCharacterColumnNamesWithColumnMappingMode(String mode)
             onDelta().executeQuery("DROP TABLE default." + tableName);
         }
     }
+
+    private void testSupportedPartitionedColumnMappingWrites(String columnMappingMode)
+    {
+        String tableName = "test_dl_dml_column_mapping_mode_" + columnMappingMode + randomNameSuffix();
+
+        onDelta().executeQuery("" +
+                "CREATE TABLE default." + tableName +
+                " (a_number INT, a_string STRING, array_col ARRAY<STRUCT<array_struct_element: STRING>>, nested STRUCT<field1: STRING>)" +
+                " USING delta " +
+                " PARTITIONED BY (a_string)" +
+                " LOCATION 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                " TBLPROPERTIES (" +
+                " delta.checkpointInterval = 1, " +
+                " delta.checkpoint.writeStatsAsJson = false, " +
+                " delta.checkpoint.writeStatsAsStruct = true, " +
+                " 'delta.columnMapping.mode'='" + columnMappingMode + "')");
```

Comment:
Why only `delta.columnMapping.mode` is single quoted?

Reply:
fixed